script/setup_docker_prereqs

```diff
@@ -1,59 +1,116 @@
 #!/bin/bash
 # Install requirements and build dependencies for Home Assinstant in Docker.
 
+INSTALL_TELLSTICK="${INSTALL_TELLSTICK:-yes}"
+INSTALL_OPENALPR="${INSTALL_OPENALPR:-yes}"
+INSTALL_FFMPEG="${INSTALL_FFMPEG:-yes}"
+INSTALL_PHANTOMJS="${INSTALL_PHANTOMJS:-yes}"
+
 # Required debian packages for running hass or components
 PACKAGES=(
+  # build-essential is required for python pillow module on non-x86_64 arch
+  build-essential
+  # libxrandr-dev is required for openalpr
+  libxrandr-dev
   # homeassistant.components.device_tracker.nmap_tracker
-  nmap net-tools
+  nmap net-tools libcurl3-dev
   # homeassistant.components.device_tracker.bluetooth_tracker
   bluetooth libglib2.0-dev libbluetooth-dev
-  # homeassistant.components.tellstick
-  libtelldus-core2
-)
-
-# Required debian packages for running hass or components from jessie-backports
-PACKAGES_BACKPORTS=(
-  # homeassistant.components.ffmpeg
-  ffmpeg
 )
 
 # Required debian packages for building dependencies
 PACKAGES_DEV=(
+  cmake git
   # python-openzwave
   cython3 libudev-dev
   # libcec
-  cmake swig libxrandr-dev
+  swig
 )
 
+install_tellstick () {
+  TELLSTICK_PACKAGES=(
+    # homeassistant.components.tellstick
+    libtelldus-core2
+  )
+
+  # Add Tellstick repository
+  echo "deb http://download.telldus.com/debian/ stable main" >> /etc/apt/sources.list.d/telldus.list
+  wget -qO - http://download.telldus.se/debian/telldus-public.key | apt-key add -
+
+  apt-get install -y --no-install-recommends ${TELLSTICK_PACKAGES[@]}
```

Comment:
You need to do a `apt-get update` since you've just added the repository. Otherwise apt can't find the `libtelldus-core2` package.